pkg/openapi/validation_test.go

```diff
@@ -46,6 +46,14 @@ func Test_ValidateMutationPolicy(t *testing.T) {
 		},
+		{
+			description: "Policy with patchesJson6902 and added element at the beginning of a list",
+			policy:      []byte(`{"apiVersion": "kyverno.io/v1","kind": "ClusterPolicy","metadata": {"name": "pe"},"spec": {"rules": [{"name": "pe","match": {"resources": {"kinds": ["Endpoints"]}},"mutate": {"patchesJson6902": "- path: \"/subsets/0/addresses/0\"\n  op: add\n  value: {\"ip\":\"123\"}\n- path: \"/subsets/1/addresses/0\"\n  op: add\n  value: {\"ip\":\"123\"}"}}]}}`),
```

Comment:
This test case currently succeeds - the jsonpatch path is `path: \"/subsets/1/addresses/0`